packages/bpk-component-theme-toggle/src/theming.js

```diff
@@ -59,7 +60,7 @@ const bpkCustomTheme = {
   buttonSecondaryTextColor: theme.primaryColor500,
   buttonSecondaryHoverTextColor: theme.primaryColor600,
   buttonSecondaryActiveTextColor: theme.primaryColor700,
-  buttonSecondaryBorderColor: theme.primaryColor500,
+  buttonSecondaryBorderColor: colorGray100,
```

Comment:
Hmm I wonder if this was intentional. @jamesf3rguson?

Reply:
Hmm, I guess its because we don't need to theme it... i.e. gray should always be gray. I guess its a bit redundant though?

Reply:
I've mentioned it in the PR description, I get it might look redundant but it's explicit, all the theme props that the button is using are there, ready to use a quick reference without relying in fallback, however if it's a deal breaker I'll remove it